src-tauri/src/telemetry_manager.rs

```diff
@@ -682,6 +687,7 @@ async fn get_telemetry_data(
         upload_speed,
         latency,
         wallet_view_key_hashed,
+        exchange_id,
```

Comment:
exhange_id should also be added to `extra_data`

The TelemetryData schema is forced on the receiving end and they don't take new fields we add. The extra_data field is dynamic. So if we add it to both we ensure they receive it.